src/index.html

```diff
@@ -9,10 +10,81 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="styles/style.css"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+    </div>
+
+    <div class="stars stars--1">
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+
+      <div class="stars_star"></div>
+    </div>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. However, do not add them between parent and child elements.